docker/vernemq/bin/vernemq.sh

```
@@ -0,0 +1,352 @@
+  echo "### Configuration file ${VERNEMQ_VM_ARGS_FILE} is empty! This will not work." >&2
+  echo "### Exiting now." >&2
+  echo "###" >&2
+  exit 1
+fi
+
+# Setup OS signal handlers
+trap 'siguser1_handler' SIGUSR1
+trap 'sigterm_handler' SIGTERM
+
+# Start VerneMQ
+/vernemq/bin/vernemq console -noshell -noinput $@ &
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)